docs/Create-Packages/Creating-a-Package.md

```diff
@@ -226,7 +226,7 @@ NuGet tracks installed solution-level packages in a `packages.config` file in th
 
 ### From a convention-based working directory
 
-Technically speaking, a NuGet package is just a ZIP file that's been renamed with the `.nupkg` extension and whose contents match certain conventions. The package's assembly DLLs, for instance, are always placed in a `lib` folder.
+Technically speaking, a NuGet package is just a ZIP file that's been renamed with the `.nupkg` extension and whose contents match certain conventions. The package's assembly DLLs, for instance, are always placed in a `lib\{tfm}` folder.
```

Comment:
Trailing slash? `lib\{tfm}\`

Reply:
dont know the diff, but wouldn't hurt adding. :)